app/code/Magento/InventoryApi/Api/ReservationAppendInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api;
+
+use Magento\InventoryApi\Api\Data\ReservationInterface;
+
+/**
+ * Domain service used to appends reservations when order is placed or canceled
+ *
```

Comment:
Yes, agreed we will use this service and RMA (Returns) would be created and handled as well